<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSql</class>
 <widget class="QWidget" name="FormSql">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutMain">
   <item>
    <layout class="QVBoxLayout" name="verticalLayoutViewTool">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutViewTool">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QToolButton" name="toolButtonFirst">
         <property name="toolTip">
          <string>Начало</string>
         </property>
         <property name="text">
          <string>Начало</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/about/icons/top.png</normaloff>:/about/icons/top.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonLast">
         <property name="toolTip">
          <string>Конец</string>
         </property>
         <property name="text">
          <string>Конец</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/about/icons/bottom.png</normaloff>:/about/icons/bottom.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="lineToolView1">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonInsert">
         <property name="toolTip">
          <string>Вставить</string>
         </property>
         <property name="text">
          <string>Вставить</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/about/icons/edit_add.png</normaloff>:/about/icons/edit_add.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonEdit">
         <property name="toolTip">
          <string>Редактировать</string>
         </property>
         <property name="text">
          <string>Редакт.</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/about/icons/edit.png</normaloff>:/about/icons/edit.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonDelete">
         <property name="toolTip">
          <string>Удалить</string>
         </property>
         <property name="text">
          <string>Удалить</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/about/icons/edit_remove.png</normaloff>:/about/icons/edit_remove.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="lineToolView2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonFilter">
         <property name="toolTip">
          <string>Фильтр</string>
         </property>
         <property name="text">
          <string>Фильтр</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/about/icons/filter.png</normaloff>:/about/icons/filter.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonRefresh">
         <property name="toolTip">
          <string>Обновить</string>
         </property>
         <property name="text">
          <string>Обновить</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/about/icons/view-refresh.png</normaloff>:/about/icons/view-refresh.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacerToolView">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QFrame" name="frameFilter">
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frameFilterTool">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,0">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="labelFilter">
            <property name="text">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Bitstream Vera Sans'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt; Фильтр:  &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonFilterAdd">
            <property name="toolTip">
             <string>Вставить строку</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/about/icons/edit_add.png</normaloff>:/about/icons/edit_add.png</iconset>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonFilterRemove">
            <property name="toolTip">
             <string>Удалить строку</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/about/icons/edit_remove.png</normaloff>:/about/icons/edit_remove.png</iconset>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonFilterApply">
            <property name="toolTip">
             <string>Применить фильтр</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/about/icons/button_ok.png</normaloff>:/about/icons/button_ok.png</iconset>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonFilterCancel">
            <property name="toolTip">
             <string>Сбросить фильтр</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/about/icons/button_cancel.png</normaloff>:/about/icons/button_cancel.png</iconset>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tableViewFilter">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>150</number>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>20</number>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
      <zorder>tableViewFilter</zorder>
      <zorder>frameFilterTool</zorder>
     </widget>
     <widget class="QTableView" name="tableViewSql">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>toolButtonLast</sender>
   <signal>clicked()</signal>
   <receiver>tableViewSql</receiver>
   <slot>scrollToBottom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonFirst</sender>
   <signal>clicked()</signal>
   <receiver>tableViewSql</receiver>
   <slot>scrollToTop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
